pkgs/tools/security/otpgen/default.nix

```
@@ -0,0 +1,40 @@
+{ 
+  pkgs
+  , lib
+  , stdenv
+  , makeWrapper
+  , fetchFromGitHub
+  , cmake
+  , pkg-config
+  , cryptopp
+  , qtbase
+}:
+
+stdenv.mkDerivation {
+
+  pname = "OTPGen";
+  version = "0.9.3";
+
+  meta = {
+    description = "Multi-purpose otp token generator written in c++ and qt with encryption and system tray icon supports totp, hotp, authy and steam and custom digit lengths and periods";
+    license = lib.licenses.mit;
+    homepage = "https://github.com/magiruuvelvet/OTPGen";
+    maintainers = with lib.maintainers; [ shamilton ];
+    platforms = stdenv.lib.platforms.linux;
+  };
+
+  nativeBuildInputs = [ cmake  ];
+  buildInputs = [ cryptopp qtbase ];
+  depsBuildBuild = [ pkg-config  ];
+
+  src = fetchFromGitHub {
+    owner = "magiruuvelvet";
+    repo = "OTPGen";
+    rev = "v0.9.3";
```

Replace `stdenv.mkDerivation {` with `stdenv.mkDerivation rec {` and you can use:

```suggestion
    rev = "v${version}";
```